src/configs/buildConfig.js

```diff
@@ -36,7 +36,7 @@ export const buildEnvironment = {
   PRISMIC_TOKEN: '_prismic_token_',
   KOCHAVA_IOS_ID: '_kochava_ios_id_',
   KOCHAVA_ANDROID_ID: '_kochava_android_id_',
-  WALLETCONNECT_PROJECT_ID: '_wallet_connect_id_',
+  WALLETCONNECT_PROJECT_ID: '15fcfb7323fcce5aa1b58afe4dc6d847',
```

Comment:
Please can we replace this with the placeholder as before

Reply:
@IAmKio DOne this change